test/fuzz/test_runner.py

```diff
@@ -152,6 +167,35 @@ def main():
         )
 
 
+def generate_corpus_seeds(fuzz_pool, build_dir, seed_dir, targets, n):
```

Comment:
`n` is unused, no?

Reply:
Yes, forgot to remove it along with the ThreadPool. Thanks!